### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240412

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1493] [Announcements]
# 应用
* 挂载网络共享：WebDAV 挂载现在支持使用 Rclone 作为挂载引擎，避免默认的 davfs2 的垃圾缓存设计（davfs2读写文件任意位置都会导致整个文件被下载，而且要整个文件下载完程序才会响应，对于大文件根本没法使用）。
  * 注意 Rclone 默认未安装，可以从 iStore 或者 “软件包” 安装，安装以后无需任何配置，也不需要启动服务，本插件只是利用了rclone可执行文件。
  * 如果需要上传（也就是挂载以后写入）超过可用内存大小的大文件，先检查本插件新增的 WebDAV 高级设置的选项。
  * 使用场景：alist 装载阿里云盘，并提供 WebDAV 服务，然后使用本插件挂载 alist 的 WebDAV 到本地，就可以让 Jellyfin 等多媒体软件扫描播放这些文件了。理论上只要 alist 支持的网盘都能通过这种方式挂载到本地。（如果挂载以后能看到阿里云盘的文件列表，但是读取文件失败（读取非阿里云盘文件时正常），有至少两个解决方法：1是修改 alist 的阿里云盘配置，将“WebDAV 策略”改成“本地代理”；2是在本插件新增的 WebDAV 高级设置里面，在“Rclone 自定义”选项里填入`--header Referer:https://www.aliyundrive.com/`）

# 设备
* rk35xx：支持部分带电池设备（例如 Nanopi R5S/R5C（电池需要额外配），Firefly Station P2 / ROC PC）的RTC时钟，断电以后可不联网恢复正常时间。 #1436
* 斐讯N1：最近支持，固件下载地址 https://fw.koolcenter.com/iStoreOS/alpha/n1/ ，刷机方式跟 N1 刷其他 linux 类型固件一致（如果刷入emmc有问题，那就先刷回官方的2.19安卓固件）。虽然放在alpha频道，实际上除了还没集成 OTA 在线升级，其他体验跟正式版无异。N1 是单网口，所以默认是DHCP客户端，没有固定ip。

**Comment by jjm2473** [comment]
N1刷机以后如果不知道怎么找IP，那么N1连接HDMI显示器和USB键盘以后再启动，这样就有终端了。等到系统启动完，终端的日志不再滚动，按一下回车键，然后执行`ip addr show dev eth0`就能看到IP地址了。或者，如果你的网络里只有N1这一个 iStoreOS 设备，也可以试试在N1的指示灯不再闪以后，直接浏览器打开 http://istoreos.lan/ ，可能成功进入后台。

**Comment by daoyelihuowang** [comment]
x86怎么更新

**Comment by jjm2473** [comment]
今天才发现上一次固件没发完，有些机型的固件都没传上去

**Comment by wanggy429** [comment]
更新后alist就彻底打不开了，请问是什么问题

**Reply by lionshuang** [reply]
我也是，重启后恢复。

**Reply by jjm2473** [reply]
前段时间作者重构了alist的包结构，可能会在alist升级以后无法启动，系统重启下就行了。参考这个 https://github.com/sbwml/luci-app-alist/issues/102

跟固件升级无关。

**Comment by qian9008** [comment]
之前更新后 首页就一直500 错误了 视图全连接不上。。。

**Reply by qian9008** [reply]
网络异常：500 connect failed

**Reply by jjm2473** [reply]
检查翻墙插件

**Comment by youhudesign** [comment]
iStoreOS的软交换小包性能目前是否想之前的openwrt一样需要使用Turbo ACC加速才能跑满，12口40g局域网

**Reply by jjm2473** [reply]
防火墙里面有flow offload，跟turboacc一样，而且是linux官方支持的。还有防火墙的FullconeNAT改成高性能模式试试。

**Comment by wanggy429** [comment]
更新后，出现无法访问后台，必须强制断电重启后恢复正常，过大概5分钟又是无法访问

**Reply by jjm2473** [reply]
你还是检查下翻墙插件或者定时任务吧

**Comment by Fengerxuanxiao** [comment]
你好，我问一下GoWebDav使用的也是Rclone 作为挂载引擎吗？

**Reply by jjm2473** [reply]
gowebdav是提供webdav服务，又不是挂载webdav，跟rclone有毛线关系。你是不是分不清挂载和提供服务。

**Reply by Fengerxuanxiao** [reply]
哈哈，我好像确实弄错了

**Comment by jonthan840923** [comment]
最近这次更新之后导致四台运行一年多稳定的机子，出现open clash无法连接的情况，中间有替换过硬盘，重刷早期版的系统测试过正常，已经排除了是机场的问题，确定是这次更新导致。太多服务了，重做系统能让人崩溃，希望下次更新能修正一下。

**Reply by jjm2473** [reply]
你说的早期版是什么版本

**Reply by jjm2473** [reply]
openclash有日志吗，开个issue提交下

**Reply by jonthan840923** [reply]
4月12日更新前一版本是好的
[OpenClash-2024-4-30-12-19-47.log](https://github.com/istoreos/istoreos/files/15158209/OpenClash-2024-4-30-12-19-47.log)

**Reply by jjm2473** [reply]
你确定这配置文件没问题？
```
  line 1: cannot unmarshal !!str `&lt;html&gt; ...` into config.RawConfig path=/tmp/yaml_sub_tmp_config.yaml
2024-04-29 13:38:50 05:38:50 ERR [Config] configuration file test failed error=yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `&lt;html&gt; ...` into config.RawConfig path=/tmp/yaml_sub_tmp_config.yaml
2024-04-29 13:38:41 05:38:41 ERR [Config] configuration file test failed error=yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `&lt;html&gt; ...` into config.RawConfig path=/tmp/yaml_sub_tmp_config.yaml
2024-04-29 12:38:25 04:38:25 ERR [Config] configuration file test failed error=yaml: unmarshal errors:
2024-04-29 12:38:13 04:38:13 ERR [Config] configuration file test failed error=rules[535] [DOMAIN-SUFFIX,6] error: proxy [6] not found path=/tmp/yaml_sub_tmp_config.yaml
```

2024-04-30的日志看起来是正常的。

**Comment by jonthan840923** [comment]
对啊，但就是连不上，4台同样问题，之前1年多都好的只做了常规在线更新

**Reply by jjm2473** [reply]
今天还是没连上？
